cmd/vela-terraform/destroy.go

```diff
@@ -117,10 +117,10 @@ func (a *Destroy) Command(dir string) *exec.Cmd {
 	if len(a.Vars) > 0 {
 		var vars string
 		for _, v := range a.Vars {
-			vars += fmt.Sprintf(" %s", v)
+			vars += fmt.Sprintf("-var=%s ", v)
 		}
-		// add flag for Vars from provided destroy command
-		flags = append(flags, fmt.Sprintf("-var=\"%s\"", strings.TrimPrefix(vars, " ")))
+		// add flag for Vars from provided validate command
```

Comment:
Is this comment change accurate?

Reply:
No - reverted the mistake from my copy/paste.